### PR TITLE
Project and About sections

### PR DESCRIPTION
**Pull request by FuadNabi**
- Add the Projects section

- Add About section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,6 +34,167 @@ <h1 class="header-h1">
         </button>
       </div>
     </header>
+    <div class="projects-cont">
```
Comment (levy002) [inline index.html]: - [x] Kindly make to wrap your project cards into the `section` tag in order to use format our HTML  semantically which will increase your website accessibility. :wink:

File: style.css [hunk 2]
```diff
@@ -66,4 +68,223 @@
     font-weight: 500;
     font-size: 16px;
   }
+
+  .start-btn:hover {
+    background-color: white;
+    color: #1a2236;
+    border: 1px solid #1a2236;
+    transition: 300ms;
+  }
```
Comment (levy002) [inline style.css:87]: - [ ] Kindly make sure to align correctly this button in order to meet the provided Figma design. :wink: 

   - ### Your design
   image


   - ### Figma design
    image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @FuadNabi   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good UI :heavy_check_mark: 
- Linters are green :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @FuadNabi,

## Status: APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 
- Great job with your design and functionalities 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**